src/test_pixel_gap_cuts.cpp

```diff
@@ -72,6 +74,12 @@ int main(int argc, char** argv) {
 
         auto cellID = sim_hit.getCellID();
         auto pos    = sim_hit.getPosition();
+        auto mom    = sim_hit.getMomentum();
+        TVector3 p; p.SetX(mom.x); p.SetY(mom.y); p.SetZ(mom.z);     
+        auto normZ  = geo.GetSensorSurface(cellID);
+    
+        double angle = normZ.Dot(p.Unit());
```

Comment:
Do you need $\arccos$ here?

Are you sure $|\mbox{normZ}|=1$?

Reply:
> Do you need arccos here?
Damn! That's was the case. TVector3 It returns Cos(angle).   
> Are you sure |normZ|=1?
No. In my local copy I changed it to normZ.Unit().

Reply:
@c-dilks, This distribution I like very much!

![image](https://github.com/eic/drich-dev/assets/52901208/339b76e0-9380-4b84-b5ad-494c04af3555)

Reply:
I thought that was more meaningful. but, I can fill the histogram before subtracting from 180 degrees.

Reply:
Angle of incidence should be $\in[0,90]^\circ$
https://en.wikipedia.org/wiki/Angle_of_incidence_(optics)

Reply:
Good to see there is a normal incidence peak though, which is what we tried to achieve when tuning the optics by eye.

Reply:
> Good to see there is a normal incidence peak though, which is what we tried to achieve when tuning the optics by eye.

![image](https://github.com/eic/drich-dev/assets/52901208/0b0c070d-c46c-49fe-be9c-8ec93273ec12)
We did a good job!